packages/flutter/test/material/chip_test.dart

```diff
@@ -254,6 +254,155 @@ void main() {
     expect(tester.getCenter(find.text('ABC')).dx, lessThan(tester.getCenter(find.byType(Icon)).dx));
   });
 
+  testWidgets('Chip responds to textScaleFactor', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      new MaterialApp(
+        home: new Material(
+          child: new Column(
+            children: <Widget>[
+              const Chip(
+                avatar: const CircleAvatar(
+                    child: const Text('A')
+                ),
+                label: const Text('Chip A'),
+              ),
+              const Chip(
+                avatar: const CircleAvatar(
+                    child: const Text('B')
+                ),
+                label: const Text('Chip B'),
+              ),
+            ],
+          ),
+        ),
+      ),
+    );
+
+    // TODO(#12357): Update this test when the font metric bug is fixed to remove the anyOfs.
```

Comment:
as in the TODO above, please provide a link rather than just a bug number

Reply:
Done.  And below.  Sorry, this is best practice internally, but I guess Github and IDEs don't link those automatically.